sponsorship_sub_management/models/sds_sub_followers.py

```diff
@@ -0,0 +1,31 @@
+##############################################################################
+#
+#    Copyright (C) 2023 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Noé Berdoz <nberdoz@compassion.ch>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+from odoo import api, models, fields
+
+
+class SdsSubFollowers(models.Model):
+    """Configuration for SUB sponsorship Notifications."""
+    _name = 'sds.sub.followers'
+    _description = 'SDS SUB Sponsorship Followers'
+
+    # Users to notify after Child Departure
+    sub_followers = fields.Many2one(
```

Comment:
The field name should be singular when it's a Many2one relation, and should be suffixed with _id by convention.
```suggestion
    user_id = fields.Many2one(
```